modules/planning/math/piecewise_jerk/BUILD

```diff
@@ -3,12 +3,12 @@ load("//tools:cpplint.bzl", "cpplint")
 package(default_visibility = ["//visibility:public"])
 
 cc_library(
-    name = "fem_1d_qp_problem",
+    name = "piecewise_jerk_problem",
```

Comment:
If we decided to fully deprecate "fem_1d_qp_problem" files, a better way is to remove the corresponding source files as well. --- If the depreciation is depending on the test result and just to change the default as "piecewise_jerk_problem", I would recommend having them both in compile and test and just use a flag to switch between which one you use. --- Either way works, but if we only remove the "piecewise_jerk_problem" from BUILD but not their source files as well. But we may accidentally leave dead code of fem_1d_qp_problem.cc/h without protection by compilation or test.